index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
```suggestion
<h2>Este es mi título <button onclick="location.reload();">Jugar otra vez</button></h2>
```
no conviene implementar un boton para actualizar la pagina